plugins/scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts

```diff
@@ -258,17 +258,9 @@ export class NunjucksWorkflowRunner implements WorkflowRunner {
           const tmpDirs = new Array<string>();
           const stepOutput: { [outputName: string]: JsonValue } = {};
 
-          if (!task.spec.metadata) {
```

Comment:
Removed this, as it's going away anyway in a follow up PR and we don't want to throw errors when it's becoming deprecated.